examples/tracer-web/README.md

```diff
@@ -1,7 +1,6 @@
 # Overview
```

Comment:
nit: it would be nice to add a screenshot that show trace/spans like we see in other examples.

Reply:
I added screenshots to plugin